readthedocs/analytics/tasks.py

```diff
@@ -77,9 +75,9 @@ def analytics_event(
 @app.task(queue='web')
 def delete_old_page_counts():
     """
-    Delete page counts older than 30 days.
+    Delete page counts older than 3 months.
 
     This is intended to run from a periodic task daily.
     """
-    thirty_days_ago = timezone.now().date() - timezone.timedelta(days=30)
-    return PageView.objects.filter(date__lt=thirty_days_ago).delete()
+    days_ago = timezone.now().date() - timezone.timedelta(days=30 * 3)
```

Comment:
I think that all of these numbers of days should be settings. So, if we decide to change them to 6 months, we have to change only one value.

Reply:
Also it needs to vary by plan, in some cases as well. But 👍  on a setting generally.

Reply:
The PR on .com overrides these tasks to take the plan into consideration.